Enterprise_Dbcache_WpdbInjection_Cluster.php

```diff
@@ -575,7 +577,7 @@ function query( $query ) {
 			if ( preg_match( '/^\s*SELECT\s+SQL_CALC_FOUND_ROWS\s/i', $query ) ) {
 				if ( false === strpos( $query, "NO_SELECT_FOUND_ROWS" ) ) {
 					$this->wpdb_mixin->timer_start();
-					$this->wpdb_mixin->_last_found_rows_result = mysqli_query( $this->wpdb_mixin->dbh, "SELECT FOUND_ROWS()" );
+					$this->wpdb_mixin->_last_found_rows_result = mysqli_num_rows( $this->wpdb_mixin->result );
```

Comment:
The result should be a `mysqli_result` and not a `string|int`, so `mysqli_query` is the correct function.  Lines 563 - 564 check if the query is "SELECT FOUND_ROWS()" and uses the cached result instead of repeating the query.

Reply:
Looking into this a bit more, I don't see any reason this needs to be a `mysqli_query` result. The only area this affects is: `is_resource( $this->wpdb_mixin->_last_found_rows_result )` which can be updated to work with `mysqli_num_rows`. I will work on this now as this is necessary to get database cluster to work as it is currently broken (no pagination).

Reply:
Looking into this a bit more, I don't see any reason this needs to be a `mysqli_query` result. The only area this affects is: `is_resource( $this->wpdb_mixin->_last_found_rows_result )` which can be updated to work with `mysqli_num_rows`. I will work on this now as this is necessary to get database cluster to work as it is currently broken (no pagination).

Reply:
ok, so the underlying issue is that `is_resource( $this->wpdb_mixin->_last_found_rows_result )` is always returning `false`. This is due to `mysqli` not using a resource, it's using objects (`mysql` legacy probably`). This now fixes pagination and still uses the `mysqli_query` which is desired.